src/test/java/io/anserini/eval/RelevanceJudgmentsTest.java

```diff
@@ -581,4 +581,355 @@ public void testMrTyDi() {
+    assertEquals(1375, getQrelsCount(qrels));
+
+    qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.mrtydi-v1.1-ru.test.txt");
+    assertNotNull(qrels);
+    assertEquals(995, qrels.getQids().size());
+    assertEquals(1168, getQrelsCount(qrels));
+
+    qrels = RelevanceJudgments.fromQrels(Qrels.MRTYDI_V11_RU_TEST);
+    assertNotNull(qrels);
+    assertEquals(995, qrels.getQids().size());
+    assertEquals(1168, getQrelsCount(qrels));
+  }
+
+  @Test
+  public void testMrTyDiSw() {
+    RelevanceJudgments qrels;
+
+    qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.mrtydi-v1.1-sw.train.txt");
+    assertNotNull(qrels);
+    assertEquals(2072, qrels.getQids().size());
+    assertEquals(2401, getQrelsCount(qrels));
+
+    qrels = RelevanceJudgments.fromQrels(Qrels.MRTYDI_V11_SW_TRAIN);
+    assertNotNull(qrels);
+    assertEquals(2072, qrels.getQids().size());
+    assertEquals(2401, getQrelsCount(qrels));
+
+    qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.mrtydi-v1.1-sw.dev.txt");
+    assertNotNull(qrels);
+    assertEquals(526, qrels.getQids().size());
+    assertEquals(623, getQrelsCount(qrels));
+
+    qrels = RelevanceJudgments.fromQrels(Qrels.MRTYDI_V11_SW_DEV);
+    assertNotNull(qrels);
+    assertEquals(526, qrels.getQids().size());
+    assertEquals(623, getQrelsCount(qrels));
+
+    qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.mrtydi-v1.1-sw.test.txt");
+    assertNotNull(qrels);
+    assertEquals(670, qrels.getQids().size());
+    assertEquals(743, getQrelsCount(qrels));
+
+    qrels = RelevanceJudgments.fromQrels(Qrels.MRTYDI_V11_SW_TEST);
+    assertNotNull(qrels);
+    assertEquals(670, qrels.getQids().size());
+    assertEquals(743, getQrelsCount(qrels));
+  }
+
+  @Test
+  public void testMrTyDiTe() {
+    RelevanceJudgments qrels;
+
+    qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.mrtydi-v1.1-te.train.txt");
+    assertNotNull(qrels);
+    assertEquals(3880, qrels.getQids().size());
+    assertEquals(3880, getQrelsCount(qrels));
+
+    qrels = RelevanceJudgments.fromQrels(Qrels.MRTYDI_V11_TE_TRAIN);
+    assertNotNull(qrels);
+    assertEquals(3880, qrels.getQids().size());
+    assertEquals(3880, getQrelsCount(qrels));
+
+    qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.mrtydi-v1.1-te.dev.txt");
+    assertNotNull(qrels);
+    assertEquals(983, qrels.getQids().size());
+    assertEquals(983, getQrelsCount(qrels));
+
+    qrels = RelevanceJudgments.fromQrels(Qrels.MRTYDI_V11_TE_DEV);
+    assertNotNull(qrels);
+    assertEquals(983, qrels.getQids().size());
+    assertEquals(983, getQrelsCount(qrels));
+
+    qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.mrtydi-v1.1-te.test.txt");
+    assertNotNull(qrels);
+    assertEquals(646, qrels.getQids().size());
+    assertEquals(677, getQrelsCount(qrels));
```

Comment:
This doesn't match the number in the Mr.TyDi paper, which says 664?
If so we should add a comment noting the difference...